src/main/java/org/spongepowered/api/text/title/Titles.java

```diff
@@ -24,17 +24,74 @@
  */
 package org.spongepowered.api.text.title;
 
+import org.spongepowered.api.text.Text;
+
 /**
  * Utility class to create instances of {@link TitleBuilder}.
  */
 public final class Titles {
 
-    private static final TitleFactory factory = null;
+    private static final Title EMPTY = new Title();
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  adam@eq2.serveftp.net
    Domain eq2.co.uk has exceeded the max emails per hour (70/60 (116%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o4.sgmail.github.com ([192.254.112.99]:24422)
	by web3.cov.yesit.co.uk with esmtps (TLSv1.2:DHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.84)
	(envelope-from <bounces+848413-d0b0-adam=eq2.co.uk@sgmail.github.com>)
	id 1YNecY-0003eG-Dg
	for adam@eq2.co.uk; Tue, 17 Feb 2015 09:39:10 +0000
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed; d=github.com; 
	h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=YxTfpUzZaHSAm/LaYdspgO96gLI=; b=jXr0TLwyBqyds0Mj
	WTSQ+HB+H9E7TNqyyexqayXSHNsk9gyl4oFAUiXbh2mkej4MzOn0J2Ql0xafjkER
	qzuCZ+r9g/cecrJMFDU9zv5p3WFkSS2tacLzZHrw/X3df9OFNsu0zY0QDj4Xqw8W
	ssP5w8YNU0q+VkZtvP2fgc947fE=
Received: by filter0066p1mdw1.sendgrid.net with SMTP id filter0066p1mdw1.9267.54E30C3AB
        2015-02-17 09:39:08.852074886 +0000 UTC
Received: from github-smtp2a-ext-cp1-prd.iad.github.net (github-smtp2a-ext-cp1-prd.iad.github.net [192.30.253.16])
	by ismtpd-005 (SG) with ESMTP id 14b96e7cd7e.6fa.1536c5
	for <adam@eq2.co.uk>; Tue, 17 Feb 2015 09:39:08 +0000 (UTC)
Date: Tue, 17 Feb 2015 01:39:08 -0800
From: gratimax <notifications@github.com>
Reply-To: SpongePowered/SpongeAPI <reply@reply.github.com>
To: SpongePowered/SpongeAPI <SpongeAPI@noreply.github.com>
Cc: Mumfrey <adam@eq2.co.uk>
Message-ID: <SpongePowered/SpongeAPI/pull/425/r24803233@github.com>
In-Reply-To: <SpongePowered/SpongeAPI/pull/425@github.com>
References: <SpongePowered/SpongeAPI/pull/425@github.com>
Subject: Re: [SpongeAPI] [WIP] Text API 2 (#425)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_54e30c3ca4c67_328e3fb39ed2729c12581d7";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: gratimax
X-GitHub-Recipient: Mumfrey
X-GitHub-Reason: comment
List-ID: SpongePowered/SpongeAPI <SpongeAPI.SpongePowered.github.com>
List-Archive: https://github.com/SpongePowered/SpongeAPI
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+001180bf3c01f1ac1c6b958f73e5a133eed97f3152ce9a9792cf0000000110face3c92a170ce017a77a1@reply.github.com>,
 <https://github.com/notifications/unsubscribe/ABGAv2H7XYBI7z1pUbZDCNj2QkxVPIo6ks5nswO8gaJpZM4DZr_C>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: adam@eq2.co.uk
X-SG-EID: VzkE0stiinYMPYWPWpnZ4OvH/Ru8MrIuYzb7JqhnFX8qro/vp73qHs5wOPJRFCiyZX79jVXC84S/UT
 wwZEqTqWLZQMTaQhA/eA9fIj2DEqBigB/pk2QSTyTa7YnD1oPkGX21dzJiRRPuzxDi2VI5pfVjqOzS
 2ElURnGMs/u0S5o=

----==_mimepart_54e30c3ca4c67_328e3fb39ed2729c12581d7
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

>  /**
>   * Utility class to create instances of {@link TitleBuilder}.
>   */
>  public final class Titles {
>  
> -    private static final TitleFactory factory = null;
> +    private static final Title EMPTY = new Title();

why are these private, and are replaced with public static methods?

---
Reply to this email directly or view it on GitHub:
https://github.com/SpongePowered/SpongeAPI/pull/425/files#r24803233
----==_mimepart_54e30c3ca4c67_328e3fb39ed2729c12581d7
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/SpongePowered/SpongeAPI/pull/425#discussion_r24803233">src/main/java/org/spongepowered/api/text/title/Titles.java</a>:</p>
<pre style='color:#555'>&gt;  /**
&gt;   * Utility class to create instances of {@link TitleBuilder}.
&gt;   */
&gt;  public final class Titles {
&gt;  
&gt; -    private static final TitleFactory factory = null;
&gt; +    private static final Title EMPTY = new Title();
</pre>
<p>why are these private, and are replaced with public static methods?</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href="https://github.com/SpongePowered/SpongeAPI/pull/425/files#r24803233">view it on GitHub</a>.<img alt="" height="1" src="https://github.com/notifications/beacon/ABGAv_TSm3aJa_NCWQqrmlDq1r4hhtg3ks5nswO8gaJpZM4DZr_C.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
  <div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
    <link itemprop="url" href="https://github.com/SpongePowered/SpongeAPI/pull/425/files#r24803233"></link>
    <meta itemprop="name" content="View Pull Request"></meta>
  </div>
  <meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_54e30c3ca4c67_328e3fb39ed2729c12581d7--